website/blog/2024-05-24-Agent/index.mdx

```diff
@@ -23,9 +23,9 @@ There are many different types of definitions of agents. When building AutoGen,
 the most generic notion that can incorporate all these different types of definitions. And to do that we
 really need to think about the minimal set of concepts that are needed.
 
-In AutoGen, we think about the agent as an entity that can act on behalf of human intent. They can send messages, receive messages, respond to other agents after taking actions and interact with other agents. We think it's a minimal set of capabilities the an agent needs to
+In AutoGen, we think about the agent as an entity that can act on behalf of human intent. They can send messages, receive messages, respond to other agents after taking actions and interact with other agents. We think it's a minimal set of capabilities that an agent needs to
 have underneath. They can have different types of backends to support them to perform
-actions and generate replies. Some of the agents can use AI models to generate replies. Some other agents can use tools
+actions and generate replies. Some of the agents can use AI models to generate replies. Some other agents can use programmed tools
```

Comment:
Sorry, I meant here to change tools to functions, so would suggest "Some other agents can use functions underneath to generate..."